content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -42,10 +42,23 @@ announce experimental support for dual stack in Istio 1.17!
 
 ## A Quick Experiment using Dual Stack
 
+{{< tip >}}
+If you want to use KinD for your test, you can setup a dual stack cluster with the following command:
```

Comment:
One small nit, sorry I missed it the first time. Otherwise LGTM.
```suggestion
If you want to use KinD for your test, you can set up a dual stack cluster with the following command:
```